docs/reference/controls/combobox.md

```diff
@@ -91,7 +92,7 @@ This example binds the items in a combo box using a data template. The C# code-b
 ```xml
 <StackPanel Margin="20">
   <ComboBox x:Name="fontComboBox" SelectedIndex="0"
-            Width="200" MaxDropDownHeight="300">
+            Width="200" MaxDropDownHeight="300" x:CompileBindings="False">
```

Comment:
Documentation shouldn't encourage users disabling compiled bindings. It's a bad (potentially app breaking) practice.